src/sentry/api/serializers/models/grouptombstone.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import absolute_import
+
+import six
+
+from sentry.api.serializers import Serializer, register, serialize
+from sentry.models import (
+    GroupTombstone, User
+)
+
+
+@register(GroupTombstone)
+class GroupTombstoneSerializer(Serializer):
+
+    def get_attrs(self, item_list, user):
+        projects = {
+            d['id']: d
```

Comment:
This seems to be a pretty common pattern in our serializers. Since the user and project serializers return a dictionary with `id`s as strings, to use `d.id`, it seems I'd have to do something like this:
```d.id: c for d, c in izip(user_list, serialize(user_list, user))```
Is that actually preferable to just doing the type conversion here?